lib/transforms/acceptance/fill-in.js

```diff
@@ -0,0 +1,44 @@
+const { makeAwait, dropAndThen, addImportStatement } = require('../../utils');
+
+/**
+ * Check if `node` is a `fillIn(selector)` expression
+ *
+ * @param j
+ * @param node
+ * @returns {*|boolean}
+ */
+function isGlobalHelperExpression(j, node) {
+  return j.CallExpression.check(node)
+    && j.Identifier.check(node.callee)
+    && node.callee.name === 'fillIn';
+}
+
+/**
+ * Transform `fillIn(selector)` to `await fillIn(selector)`, remove `andThen` calls
+ *
+ * @param file
+ * @param api
+ * @returns {*|string}
+ */
+function transform(file, api) {
+  let source = file.source;
+  let j = api.jscodeshift;
+
+
+  let root = j(source);
+
```

Comment:
The `:eq(2)` optimization I mention is probably common to any interaction helper.